mtp_bank_admin/apps/bank_admin/views.py

```diff
@@ -86,6 +101,8 @@ def download_bank_statement(request, receipt_date):
         response = HttpResponse(bai2, content_type='application/octet-stream')
         response['Content-Disposition'] = 'attachment; filename="%s"' % filename
 
+        logger.info('User "%s" is downloading bank statement file' % request.user.username)
```

Comment:
Include the receipt_date that they're downloading the file for on this one as well?